installer.sh

```diff
@@ -39,9 +44,22 @@ has_sudo=""
 kernel=""
 machine=""
 version=""
+package="platformsh-cli"
+docs_url="https://docs.platform.sh"
+support_url="https://platform.sh/support"
+
+if [ "$VENDOR" == "upsun" ]; then
+    BREW_FORMULA="platformsh/tap/upsun-cli"
+    binary="upsun"
+    vendor_name="Upsun"
+    package="upsun-cli"
+    docs_url="https://docs.upsun.com"
+    support_url="https://console.upsun.com/support"
```

Comment:
This link doesn't work for me? And it requires authentication?

we should make an upsun.com/support (similar to platform.sh/support) - but in this case it might want to be something like the platform.sh/contact page?

Reply:
```suggestion
```

Reply:
I'll leave the Platform.sh contact page, as we don't have a separate one for Upsun. Also, at the end of the day the same people will receive the message.